### DISCUSSION TITLE
Проблема с YouTube

### DISCUSSION THREAD

**Discussion by MascotWorld** [xtrime-ru/antizapret-vpn-docker#75] [Q&A]
У меня почему-то shorts на телефоне через раз грузит (при этом на ПК все ок) + у меня стоит google nest hub 2nd и там такая же проблема только с видео(может загрузить, а может уйти в бесконечную загрузку). Может как-то логи собрать чтобы исправить?)

**Comment by xtrime-ru** [comment]
Расширенное логирование DNS запросов включается через переменную LOG_DNS=1 в docker-compose.

После изменения переменной:
```
docker compose up -d
docker compose logs -f antizapret-vpn | tee az_logs.txt
```

После этого запустить short и подождать сбора логов. Они сохранятся в файл az_logs.txt

**Comment by shilophrenic** [comment]
> У меня почему-то shorts на телефоне через раз грузит (при этом на ПК все ок) + у меня стоит google nest hub 2nd и там такая же проблема только с видео(может загрузить, а может уйти в бесконечную загрузку). Может как-то логи собрать чтобы исправить?)

Не только у тебя такая проблема. У меня на iPhone такая же беда
